contrib/etc/ebusd/mqtt-hassio.cfg

```diff
@@ -105,7 +105,7 @@ filter-seen = 5
 #filter-non-circuit =
 # include only messages having the specified name (partial match, alternatives and wildcard supported).
 # HA integration: filter to some useful names for monitoring the heating circuit
-filter-name = status|temp|humidity|yield|count|energy|power|runtime|hours|starts|mode|curve|^load$|^party$|sensor|timer
+filter-name = status|temp|humidity|yield|count|energy|power|runtime|hours|starts|mode|curve|^load$|^party$|sensor|timer|co2|air|flow|filter|inlet|exhaust|pressure|reset
```

Comment:
where are the CSVs matching these changes?

Reply:
@john30 The files are waiting in [this PR](https://github.com/john30/ebusd-configuration/pull/406).